rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -275,6 +276,30 @@ impl<'me, K: ArrowReadableKey<'me> + Into<KeyWrapper>, V: ArrowReadableValue<'me
         None
     }
 
+    pub(super) async fn load_blocks(&self, block_ids: Vec<Uuid>) -> () {
+        let mut futures = Vec::new();
+        for block_id in block_ids {
+            futures.push(self.get_block(block_id));
+        }
+        join_all(futures).await;
```

Comment:
I think this should at some point be a task

Reply:
Yes that is a separate PR. I added a TODO